docs/en/sql-reference/statements/create/table.md

```diff
@@ -548,25 +550,34 @@ It’s possible to use tables with [ENGINE = Memory](../../../engines/table-engi
 
 ## REPLACE TABLE
 
-'REPLACE' query allows you to update the table atomically.
+The `REPLACE` statement allows you to update a table [atomically](/docs/en/concepts/glossary#atomicity).
 
 :::note
-This query is supported only for [Atomic](../../../engines/database-engines/atomic.md) database engine.
+This statement is supported for the [`Atomic`](../../../engines/database-engines/atomic.md) and [`Replicated`](../../../engines/database-engines/replicated.md) database engines, 
+which are the default database engines for ClickHouse and ClickHouse Cloud respectively.
 :::
 
-If you need to delete some data from a table, you can create a new table and fill it with a `SELECT` statement that does not retrieve unwanted data, then drop the old table and rename the new one:
+Ordinarily, if you need to delete some data from a table, 
+you can create a new table and fill it with a `SELECT` statement that does not retrieve unwanted data, 
+then drop the old table and rename the new one. 
+This approach is demonstrated in the example below:
 
 ```sql
 CREATE TABLE myNewTable AS myOldTable;
-INSERT INTO myNewTable SELECT * FROM myOldTable WHERE CounterID <12345;
+INSERT INTO myNewTable
```

Comment:
There's more than one sql statement in this snippet, keeping each statement in it's own line and not splitting it out onto new lines would be less confusing to read - similar comment for other snippets below.